#!/usr/bin/env python
PACKAGE = "cnbiros_shared_navigation"

import math
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angle_min",	double_t, 0, "Minimum grid angle", -120.0 , -120.0, 0.0)
gen.add("angle_max",	double_t, 0, "Maximum grid angle",  120.0 , 0.0, 120.0)
gen.add("angle_inc",	double_t, 0, "Increment grid angle", 9.0 , 0.1, 120.0)
gen.add("range_min",	double_t, 0, "Minimum grid range", 0.0, 0.0, 10.0)
gen.add("range_max",	double_t, 0, "Maximum grid range",  6.0, 0.0, 10.0)
gen.add("publish_frequency", double_t, 0,  "Publish frequency of the node",	20, 0.1, 100)

exit(gen.generate(PACKAGE, "laserscan_grid_parameters", "LaserScanGrid"))
